requirements.txt

```diff
@@ -3,4 +3,5 @@ Django==4.1.10
 djangorestframework==3.13.1
 pytz==2022.1
 sqlparse==0.4.4
-# mysqlclient==1.4.3
\ No newline at end of file
+# mysqlclient==1.4.3
+mysql-connector-python==8.2.0
```

Comment:
Fix EoF

Reply:
Can't understand
just remove line # mysqlclient==1.4.3?

Reply:
You need to add a new line at the end of file

Reply:
End of the File - oh my god. Soryy